numpy/add_newdocs.py

```diff
@@ -2265,25 +2265,86 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """)
 
+add_newdoc('numpy.core', 'vdot',
+    """
+    vdot(a, b)
+
+    Return the dot product of two vectors.
+
+    The vdot(`a`, `b`) function handles complex numbers differently than
+    dot(`a`, `b`).  If the first argument is complex the complex conjugate
+    of the first argument is used for the calculation of the dot product.
+
+    Note that `vdot` handles multidimensional arrays differently than `dot`:
```

Comment:
It could be cleaner to cast this discussion in terms of inner products like `i,i->` and `ij,ij->`. I think the inner product language may be slightly more universal.

Reply:
I agree, but actually I didn't amend this docstring. It was thought I should remove this sec but then advised to re-add, and when I did I inadvertently changed the order of functions, so this `add_newdoc` `vdot` now comes before `c_einsum` instead of after, the corresponding deletions are below, unchanged. I would prefer to just leave this as is. For clarity I have only actually altered the `einsum` docstring.

Reply:
Works for me, I think this ready for merge!